scripts/metricsgen/metricsgen.go

```diff
@@ -168,7 +168,7 @@ func ParseMetricsDir(dir string, structName string) (TemplateData, error) {
 
 	// Grab the package name.
 	var pkgName string
-	var pkg *ast.Package //nolint:staticcheck
+	var pkg *ast.Package
```

Comment:
Apparently it fails with and without `//nolint` here 🤦

Reply:
nvm, it's failing again